### ISSUE TITLE
[Feature Request] Support for Delivery Groups

### ISSUE THREAD

**Issue by Veldkornet** [PandaNL/nl.tonversteeg.pushover#9]
Could you please add support for delivery groups?

I have a workaround of just adding the delivery group key to the settings, but then of course you can no longer send messages to specific devices. 

Currently you have two cards:
Pushover Notification
Pushover Notification (Device)

My thought was to just add a third card:
Pushover Notification (Group)

This would work similar to devices where you can then select one of the groups that you belong to.

**Comment by dannymertens** [comment]
Also missing this, please can you add this? Thanks!

**Comment by Veldkornet** [comment]
FYI, what I have done here is set the notification group as the default in settings which is fine in most cases, except then the device notification doesn’t work.

**Comment by Veldkornet** [comment]
I seem to have got this working. I added an extra field in the settings to allow you to add the group key.
Then in the Send Notification and Send Image notification cards, I've added the ability to select to send the notification to a user or a group.

Of course, this makes no sense to add it to the Send to Device card, so I've not added the options there.

My own testing looks good an everything is working, therefore I've put in a pull request for this.
